api/src/main/java/org/openmrs/Drug.java

```diff
@@ -276,4 +278,26 @@ public void addDrugReferenceMap(DrugReferenceMap drugReferenceMap) {
 			getDrugReferenceMaps().add(drugReferenceMap);
 		}
 	}
+	
+	/**
+	 * Gets doseLimitUnits field to Drug, which will represent the units of the existing fields of
```

Comment:
Given that this is already the Dug class, what value does Drug add to the decription?